packages/yew/src/html/component/properties.rs

```diff
@@ -11,20 +11,106 @@ pub trait Properties: PartialEq {
     fn builder() -> Self::Builder;
 }
 
-/// Builder for when a component has no properties
-#[derive(Debug, PartialEq)]
-#[doc(hidden)]
-pub struct EmptyBuilder;
+mod macro_export {
```

Comment:
We use the convention `__macro` for modules/functions which are only for macros. This also needs to be `#[doc(hidden)]`